src/request/components/request-detail-panel-execution.jsx

```diff
@@ -216,15 +392,28 @@ module.exports = React.createClass({
             var beforeViewComponentMessages = message.beforeViewComponent;
             var afterViewComponentMessages = message.afterViewComponent;
             var afterActionResultMessage = message.afterActionResult;
+                                
             if (beforeExecuteCommandMessages && afterExecuteCommandMessages) {
                 beforeExecuteCommandMessages = beforeExecuteCommandMessages.sort(function(a, b) { return a.ordinal - b.ordinal; });
                 afterExecuteCommandMessages = afterExecuteCommandMessages.sort(function(a, b) { return a.ordinal - b.ordinal; });
             }
+
+            // combine and sort all mongoDB messages
+            var mongoDBMessages = [];
+            var m = [message.dataMongodbInsert, message.dataMongodbRead, message.dataMongodbUpdate, message.dataMongodbDelete];
+            for ( var i = 0; i<m.length;i++) {
+                if ( m[i] && m[i].length > 0) {
+                    mongoDBMessages = mongoDBMessages.concat(m[i]);
+                }
+            }
+            if ( mongoDBMessages) {
+                mongoDBMessages = mongoDBMessages.sort(function(a,b) { return a.ordinal - b.ordinal;});
+            }
```

Comment:
Can the above be put into a function? Just cleans things up a little... even though its changing later on. It helps isolate the logic and whats going on as future changes come in.